Session-6/DeckOfCards/CardsDeck.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace DeckOfCards
+{
+    public abstract class CardsDeck<T>
```

Comment:
This class shouldn't be for any type. This should be applicable only for the ICard types